### PR TITLE
fhc cordova migrate command

### PR DESCRIPTION
**Pull request by corinnekrych**
@aidenkeating review?

### CODE REVIEW

**Comment by corinnekrych** [comment]
Not sure where to put the test @aidenkeating where would you do?

**Comment by aidenkeating** [comment]
@corinnekrych Maybe `test/unit/fh3/cordova/test_migrate.js`, it should just follow the same structure as all others. If you'd like I can help out for this part?

**Comment by corinnekrych** [comment]
@aidenkeating let me add it

File: test/unit/fh3/cordova/migrate_test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+var migrate = genericCommand(require('cmd/fh3/cordova/migrate'));
+module.exports = {
+  setUp : function(cb){
+    return cb();
+  },
+  'test cordova migrate': function(cb) {
+    migrate({app : '1a'}, function (err, data){
+      assert.equal(err, null, err);
+      assert(data.ok === true);
+      return cb();
```
Comment (aidenkeating) [inline test/unit/fh3/cordova/migrate_test.js]: I think this could just be `assert(data.ok)`

**Comment by aidenkeating** [comment]
👍 Will merge once the build completes

**Pull request merged**